README.md

```diff
@@ -2192,6 +2192,7 @@ _**Unofficial** set of patterns for structuring projects._
 * [hipchat (xmpp)](https://github.com/daneharrigan/hipchat) - A golang package to communicate with HipChat over XMPP.
 * [igdb](https://github.com/Henry-Sarabia/igdb) - Go client for the [Internet Game Database API](https://api.igdb.com/).
 * [kanka](https://github.com/Henry-Sarabia/kanka) - Go client for the [Kanka API](https://kanka.io/en-US/docs/1.0).
+* [lark](https://github.com/chyroc/lark) - [Feishu](https://open.feishu.cn/)/[Lark](https://open.larksuite.com/) Open API Go SDK, Support ALL Open API and Event Callback.
```

Comment:
Please add more comments (docs) https://goreportcard.com/report/github.com/chyroc/lark#golint

Reply:
hi, @phanirithvij ，i add some comment，and go lint result is 87%

![image](https://user-images.githubusercontent.com/15604894/146710113-47d6f327-cd4f-4c6c-a806-ea591d79fbb5.png)